### ISSUE TITLE
Extract frames from the Moments in Time dataset

### ISSUE THREAD

**Issue by fonsc** [thefonseca/predictive-coding#1]
The PredNet model uses sequences of images as input and we need to extract this information from the Moments in Time video dataset. More specifically, we will focus on the Moments in Time "Mini" version. The links are listed below:

> The cached raw data can be downloaded from the following link (305 GB), http://data.csail.mit.edu/soundnet/actions3/split1/Moments_in_Time_Raw.zip.

> A pre-processed, 256x256 resolution and 30 fps, cached version of the data can be downloaded from the following link (73 GB), http://data.csail.mit.edu/soundnet/actions3/split1/Moments_in_Time_256x256_30fps.zip.

> We also provide a subset of our videos, Moments in Time Mini, that contains 500 videos per class for 200 classes (9.4 GB), http://data.csail.mit.edu/soundnet/actions3/split1/Moments_in_Time_Mini.zip.

> The test set for the full track can be downloaded from [here](http://data.csail.mit.edu/soundnet/actions3/split1/test.zip), and the test set for the mini track can be downloaded from [here](http://data.csail.mit.edu/soundnet/actions3/split1/momentsMiniTest.zip).

Other related resources:

> We are hosting a challenge at CVPR 2018, jointly with the ActivityNet Challenge, on both the full Moments in Time dataset which is open to the public and the Moments in Time Mini dataset which is open to students.  More information can be found here, http://moments.csail.mit.edu/challenge.html.

> All cached videos (unprocessed) can also be manually downloaded from http://data.csail.mit.edu/soundnet/actions3/.
For example, "http://data.csail.mit.edu/soundnet/actions3/dancing/yt-5of6ffo6-Bw_205.mp4" for the training video "dancing/yt-5of6ffo6-Bw_205.mp4".

> We provide a GitHub repository with pre-trained models and example code for using the models, located here https://github.com/metalbubble/moments_models.

> We ask that anyone that uses the data, or pre-trained models, please cite this paper https://arxiv.org/abs/1801.03150.

**Comment by sudonto** [comment]
Hi @thefonseca , do you have the labels for **test set** of MIT mini track?

**Comment by thefonseca** [comment]
> Hi @thefonseca , do you have the labels for **test set** of MIT mini track?

Sorry @sudonto, I'm afraid I don't have it. I suggest you contact the [authors](http://moments.csail.mit.edu) if you need them.

**Comment by yintao-xu** [comment]
> 

Hi,i wanna download http://data.csail.mit.edu/soundnet/actions3/split1/Moments_in_Time_Mini.zip.
But the link is invalid.
I didn't find any results on the Internet.
What should I am going to do?

**Comment by sudonto** [comment]
@yintao-xu , please dm me.

**Comment by yintao-xu** [comment]
> @yintao旭，请dm我。

What does DM mean？
Can you download the file Moments_in_Time_Mini.zip？

**Comment by sudonto** [comment]
updated

**Comment by yintao-xu** [comment]
> 

i send email to notifications@github.com.
my email is 1965118240@qq.com

**Comment by meisa233** [comment]
I can't download it either, can someone give me a valid link?
